Terminal.Gui/Views/ListView.cs

```diff
@@ -857,7 +908,24 @@ public Task SetSourceAsync (IList source)
                                      );
     }
 
-    private void ListView_LayoutStarted (object sender, LayoutEventArgs e) { EnsureSelectedItemVisible (); }
+    private void ListView_DrawAdornments (object sender, DrawEventArgs e) { SetScrollRowsColsSize (); }
```

Comment:
I tried using `LayoutComplete` and something else but without success. Why didn't it work? Because the adornments are drawn first and the scroll bar needs all the information provided by the `Parent`, before being drawn. For this reason I created the `DrawAdornments` event to be called by the `Parent` before the scroll bar and the parent are drawn. It was the only solution I found and I think it's not that hacky because this is done before the `Parent` is drawn.
Furthermore, you know that `LayoutComplete` is normally called when there is a resizing of a view and depends on the `LayoutNeeded` variable which is cleared after being executed. It wasn't really working in every situation where I resized the terminal and therefore didn't adjust the content well.